nautobot/extras/api/serializers.py

```diff
@@ -1037,6 +1048,40 @@ class Meta:
             "last_updated",
         ]
 
+    def validate(self, data):
+        data = super().validate(data)
+
+        # All relevant content_types should be assigned to tag for API Version <1.3
+        data["content_types"] = TaggableClassesQuery().as_queryset
+
+        return data
+
+
+class TagSerializerVersion13(CustomFieldModelSerializer):
+    url = serializers.HyperlinkedIdentityField(view_name="extras-api:tag-detail")
+    tagged_items = serializers.IntegerField(read_only=True)
+    content_types = ContentTypeField(
+        queryset=TaggableClassesQuery().as_queryset,
+        many=True,
+        required=True,
+    )
+
+    class Meta:
+        model = Tag
+        fields = [
+            "id",
+            "url",
+            "name",
+            "slug",
+            "color",
+            "description",
+            "tagged_items",
+            "content_types",
+            "custom_fields",
+            "created",
+            "last_updated",
+        ]
```

Comment:
Can we reduce code duplication in any way by having `TagSerializer` and `TagSerializerVersion13` inherit from a common parent class?

Reply:
See: https://github.com/nautobot/nautobot/pull/1420/files#diff-7be95518aa8bbb11712e2bbee251ee299ad56b9a9eefea9a130fde983cdf8271L390-R446